packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -9,26 +9,46 @@ import { Strings } from '../constants'
 import { WebviewRef } from '../models/environments'
 
 export class NativeService {
-  private messengerService = NativeMessenger
+  private messengerService: typeof NativeMessenger
```

Comment:
my IDE suggests me to:
```suggestion
  private readonly messengerService: typeof NativeMessenger
```

Reply:
Thanks, replaced in https://github.com/cozy/cozy-libs/compare/18c6fb9c2fc90ac667c8e10b42f29552f2dd360b..c8724f304b52d9a21b6cb7a9aea1ce0165e236e9